.github/workflows/versionci.yml

```diff
@@ -0,0 +1,20 @@
+name: Version CI
+on:
+  pull_request:
+  push:
+    branches:
+      - main
```

Comment:
What's the benefit or reasoning behind using GitHub actions to run tests on the various Ruby versions rather than Travis, which is already configured on the repo?

Travis matrix testing: https://docs.travis-ci.com/user/build-matrix/

Reply:
Since this is just a simple gem, it does feel like overkill to set up Travis to just run rspec. I can get on board with using GitHub actions.